Jenkinsfile.snyk

```diff
@@ -1,4 +1,5 @@
-@Library('csm-shared-library') _
+//@Library('csm-shared-library') _
+@Library('csm-shared-library@feature/publish-snyk-reports') __
```

Comment:
Temporary change, needed for testing. before this PR is merged, it needs to be rolled back.